docs/integrations/observability/_index.rst

```diff
@@ -193,4 +193,80 @@ Monitoring Dashboard
+   the password you set above. You should visit the ``Determined API Server Monitoring`` dashboard.
+
+******************
+ Metrics
+******************
+Determined does not produce metrics, rather it uses existing tools to report information. Visit 
+source documentation for details on specific metrics.
+
+API Performance Metrics
+=======================
+
+Determined master reports API performance metrics using `grpc ecosystem <https://github.com/grpc-ecosystem/go-grpc-prometheus?tab=readme-ov-file#metrics>`__.
+
+Kubernetes and Container Metrics
+================================
+
+The kube-prometheus-stack enables kube-state-metrics and cAdvisor by default. 
+
+-  `kube-state-metrics <https://github.com/kubernetes/kube-state-metrics/tree/main/docs#exposed-metrics>`__ reports the state of kubernetes objects, including Determined and any resources it creates
+-  `cAdvisor <https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md>`__ reports resource usage and performance of running containers, for example memory and cpu used
+
+Nvidia DCGM Exporter
+====================
+
+Nvidia's Data Center GPU Manager collects data on Nvidia GPUs. 
+
+-  by default, only the most useful `subset <https://github.com/GoogleCloudPlatform/prometheus-engine/blob/8dd8a187486cccb5ede3132e5773ae786239dbc2/examples/nvidia-dcgm/exporter.yaml#L139-L169>`__ of metrics are scraped by Prometheus
+-  the full list of metrics generated by DCGM exporter can be found `here <https://github.com/NVIDIA/dcgm-exporter/blob/main/etc/dcp-metrics-included.csv>`__
+
+Health Status
+=============
+
+Determined master reports a metric, ``determined_healthy``, with value ``1`` when major dependencies 
+are reachable and ``0`` otherwise.  Visit `alerting <https://github.com/determined-ai/determined/blob/observability_feature_branch/docs/integrations/prometheus/_index.rst#alerts>`__ 
+for information on how to set up alerts.
+
+Viewing Metrics
+===============
+
+The Determined Master labels pods it creates with Determined state values. The following pod labels 
+can be accessed via ``kube_pod_labels`` metric from kube-state-metrics. Label names will be formatted 
+as ``label_determined_ai_<label_name>``, e.g. ``label_determined_ai_container_id``. 
+
+.. list-table:: Determined Pod Labels
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Label Key
+     - Label Value Description
+   * - determined.ai/container_id 
+     - 
+   * - determined.ai/experiment_id
+     - ``task_type=TRIAL`` only
+   * - determined.ai/resource_pool
+     - name of the resource pool, including ``default`` 
+   * - determined.ai/task_id 
+     - 
+   * - determined.ai/task_type
+     - Determined task type, e.g. ``TRIAL``, ``NOTEBOOK``, ``TENSORBOARD``
+   * - determined.ai/trial_id
+     - ``task_type=TRIAL`` only 
+   * - determined.ai/user
+     - Determined username that initiated the request 
+   * - determined.ai/workspace
+     - name of the workspace, including ``Uncategorized``
+
```

Comment:
this table breaks the docs build

use a table editor like https://tableconvert.com/restructuredtext-generator

run `make fmt` then `make live`